template/base/base.html

```diff
@@ -112,32 +112,18 @@
         </div>
 
         <div class="navbar-collapse collapse" id="navbar-mobile">
+            <!--
             <ul class="nav navbar-nav">
                 <li>
-                    <a class="sidebar-control sidebar-main-toggle hidden-xs">
+                    <a href="/">
                         <i class="icon-paragraph-justify3"></i>
                     </a>
                 </li>
-
-                <li><a href="#">Text link</a></li>
-
-                <li>
-                    <a href="#">
-                        <i class="icon-calendar3"></i>
-                        <span class="visible-xs-inline-block position-right">Icon link</span>
-                    </a>
-                </li>
             </ul>
+            -->
```

Comment:
is this part(115-123) commented out? If commented out, what's the change at 118 for?

Reply:
Changed 118 first to properly link the icon - but then decided to comment the block out for future use because it serves no function for now (no new content to link to)